omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -290,6 +290,18 @@
     end
   end
 
+  block "Remove .pyi files" do
+    # We don't need them at runtime
+
+    pattern = "#{install_dir}/embedded/lib/python#{python_version}/site-packages/**/*.pyi"
```

Comment:
Should we also remove `py.typed` files? I think they're empty but they exist exclusively for type checker use, and since we're removing type information...

Reply:
Also addressed here https://github.com/DataDog/datadog-agent/pull/34496/commits/bf2a757c96e17d1ea1bc743d36af41d66caaa8f1 What do you think?